src/widgets/mentors/ui/mentors.tsx

```diff
@@ -13,13 +13,15 @@ export const Mentors = () => {
     <section className="mentors container" id="mentors-wanted">
       <div className="mentors content column-2">
         <div className="mentors-info">
-          <WidgetTitle size="large" mods="lines">Mentors wanted!</WidgetTitle>
-          <Subtitle
-            text="The Rolling Scopes School is constantly looking for mentors from all over the world to
+          <WidgetTitle size="large" mods="lines">
+            Mentors wanted!
+          </WidgetTitle>
+          <Subtitle>
```

Comment:
I think we can clean '.subtitle' styles:

![image](https://github.com/user-attachments/assets/aa8f0619-3262-41b8-a82e-be7ca05291f5)